docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/channels-reference-guide.md

```diff
@@ -24,35 +24,35 @@ Users can enable a workflow for [buyers and sellers](../../../orders-and-fulfill
 
 Users can set the [Shipping Tax Category](../../../store-administration/configuring-taxes/creating-tax-categories.md), the Price Type, and the Discount Target Price Type.
 
-For _Price Type_, users can determine whether a price list or promotion is calculated using the gross price or the net price.
-
-For _Discounts Target_, users can determine whether applicable discounts are calculated based on the product's gross price or net price.
+For _Price Type_, users can determine whether product prices are displayed with either tax included (gross price) or tax excluded (net price).
 
 ![The Prices tab is where the tax categories and prices are set.](./channels-reference-guide/images/03.png)
 
 ### Health Checks
 
-Health checks allow users to quickly fix any issues with their store sites, such as missing widgets.
+Health checks allow users to quickly fix any common setup issues with their store sites, such as missing widgets that are required to have a functioning store.
 
 ![Health checks allow you to quickly add Commerce portlets or fix issues if you had built the site manually.](./channels-reference-guide/images/04.png)
 
 ### Payment Methods
 
-Users can configure which [payment methods](../../../store-administration/configuring-payment-methods/managing-payment-methods.md) to use for the store. In addition, users can use a [custom payment engine](../../../developer-guide/tutorials/implementing-a-new-payment-method.md).
```

Comment:
Watch out with the terms site, channel and store. They have specific meanings. In this case, the configuration is of the channel, not of the site.

Reply:
Fixed